errors/url-deprecated.mdx

```diff
@@ -35,4 +33,4 @@ export default withRouter(Page)
 
 We provide a codemod (a code to code transformation) to automatically change the `url` property usages to `withRouter`.
 
-You can find this codemod and instructions on how to run it here: /docs/advanced-features/codemods#url-to-withrouter
+You can find this codemod and instructions on how to run it here: [Use `withRouter`](docs/pages/building-your-application/upgrading/codemods#url-to-withrouter)
```

Comment:
```suggestion
You can find this codemod and instructions on how to run it here: [Use `withRouter`](/docs/pages/building-your-application/upgrading/codemods#url-to-withrouter)
```

Reply:
Thank you for cheking my typo.
I updated